modeling/game_ends_and_starts_again.plantuml

```diff
@@ -0,0 +1,52 @@
+@startuml
+
+title Game ends and starts again with 2 players
```

Comment:
Collect all sequence diagrams into one and then seperate different scenarios with `note across: game ends`